packages/components/src/components/link/test/html.mock.ts

```diff
@@ -10,20 +9,12 @@ export const getLinkHtml = (props: LinkProps, innerHTML = ''): string => {
 		{
 			_href: 'javascript:void(0)',
 			_hideLabel: false,
-			_label: '',
-			// _label: '…', // ⚠ required
 			_tooltipAlign: 'right',
 			_targetDescription: 'Der Link wird in einem neuen Tab geöffnet.',
 		},
 		props
 	);
-	if (typeof props._ariaLabel === 'string' && isEmptyOrPrefixOf(props._label, props._ariaLabel) === false) {
-		if (props._label.length > 0) {
-			props._ariaLabel = props._label;
-		} else {
-			props._label = props._ariaLabel;
-		}
-	}
+	const label = typeof props._label === 'string' && props._label.length >= 3 ? props._label : props._href;
```

Comment:
Diese Condition muss anders formuliert werden, weil die eigentlich Validierung von Label erfolgt im validateLabel. Ja vielleicht ist hier dann eine Prüfung auf `this._label` sinnvoll.

Reply:
siehe oben